firmware/common/portapack_persistent_memory.cpp

```diff
@@ -269,7 +269,7 @@ struct data_t {
           frequency_rx_correction(0),
           frequency_tx_correction(0),
 
-          encoder_dial_sensitivity(0),
+          encoder_dial_sensitivity(2),
```

Comment:
I would recommend using the enum "DIAL_SENSITIVITY_NORMAL" as on line 366. 
(I was under the impression that the "defaults()" function on line 361 would initialize these, but good idea to set it here as well.)

Reply:
I must have been tired. Thanks, I replaced it ;-)